src/overlays/actors/ovl_En_Wiz_Brock/z_en_wiz_brock.c

```diff
@@ -28,19 +36,109 @@ const ActorInit En_Wiz_Brock_InitVars = {
     (ActorFunc)EnWizBrock_Draw,
 };
 
-#endif
+void EnWizBrock_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnWizBrock* this = THIS;
+    CollisionHeader* colHeader = NULL;
+    s32 pad;
+
+    DynaPolyActor_Init(&this->dyna, 0);
+    CollisionHeader_GetVirtual(&object_wiz_Colheader_001690, &colHeader);
+    this->dyna.bgId = DynaPoly_SetBgActor(globalCtx, &globalCtx->colCtx.dyna, &this->dyna.actor, colHeader);
+    this->dyna.actor.colChkInfo.mass = MASS_IMMOVABLE;
+    this->dyna.actor.colChkInfo.health = 3;
+    this->unk1A6 = 0;
+    Actor_SetScale(&this->dyna.actor, 0.01f);
+    this->platformNum = platformCount++;
+    this->actionFunc = EnWizBrock_SetupUpdateStatus;
+    this->dyna.actor.scale.x = this->dyna.actor.scale.y = this->dyna.actor.scale.z = 0.01f;
+    this->alpha = 255.0f;
+}
+
+void EnWizBrock_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+    EnWizBrock* this = THIS;
+
+    DynaPoly_DeleteBgActor(globalCtx, &globalCtx->colCtx.dyna, this->dyna.bgId);
+}
+
+void EnWizBrock_SetupUpdateStatus(EnWizBrock* this, GlobalContext* globalCtx) {
+    this->actionFunc = EnWizBrock_UpdateStatus;
+}
+
+/**
+ * @brief Checks the platform status, when the Wizzrobe is defeated, starts incrementing despawnCounter to
+ * 0x25 at which point the platforms are despawned.
+ */
+void EnWizBrock_UpdateStatus(EnWizBrock* this, GlobalContext* globalCtx) {
+    if (this->platformType == PLATFORM_TYPE_INACTIVE) {
+        if (this->dyna.actor.colChkInfo.health != 3) {
+            this->platformType = this->dyna.actor.colChkInfo.health;
+        }
+    }
+
+    if (this->dyna.actor.colChkInfo.health == 0) {
+        this->despawnCounter++;
+        if ((gGameInfo->data[0x991] + 0x1E) < this->despawnCounter) {
```

Comment:
These should use the REG macros; there's a script to convert them. Also, dec again.